cicd/builds/triggering-builds-build-hooks.adoc

```diff
@@ -12,6 +12,13 @@ include::modules/builds-triggers.adoc[leveloffset=+1]
 
 include::modules/builds-webhook-triggers.adoc[leveloffset=+2]
 
+include::modules/unauthenticated-users-system-webhook.adoc[leveloffset=+3]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../authentication/using-rbac.adoc#unauthenticated-users-cluster-role-bindings-concept_using-rbac[Adding unauthenticated groups to cluster roles]
```

Comment:
The link text doesn't match the link destination. 

![image](https://github.com/openshift/openshift-docs/assets/25209155/4da39003-ba4a-444f-9080-7eaf40a5bc92)

Also, should the link be
```suggestion
* xref:../../authentication/using-rbac.adoc#unauthenticated-users-cluster-role-bindings-con_using-rbac[Adding unauthenticated groups to cluster roles]
```